<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1293</width>
    <height>875</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1291</width>
      <height>871</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="MainFrame">
     <property name="leftMargin">
      <number>4</number>
     </property>
     <property name="topMargin">
      <number>4</number>
     </property>
     <property name="rightMargin">
      <number>4</number>
     </property>
     <property name="bottomMargin">
      <number>4</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="Cmd">
       <item>
        <layout class="QVBoxLayout" name="LoadLayout">
         <item>
          <widget class="QLabel" name="LoadLbl">
           <property name="text">
            <string>Загрузка</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="FileNameEntry">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="palette">
            <palette>
             <active>
              <colorrole role="PlaceholderText">
               <brush brushstyle="SolidPattern">
                <color alpha="128">
                 <red>113</red>
                 <green>113</green>
                 <blue>113</blue>
                </color>
               </brush>
              </colorrole>
             </active>
             <inactive>
              <colorrole role="PlaceholderText">
               <brush brushstyle="SolidPattern">
                <color alpha="128">
                 <red>113</red>
                 <green>113</green>
                 <blue>113</blue>
                </color>
               </brush>
              </colorrole>
             </inactive>
             <disabled>
              <colorrole role="PlaceholderText">
               <brush brushstyle="SolidPattern">
                <color alpha="128">
                 <red>113</red>
                 <green>113</green>
                 <blue>113</blue>
                </color>
               </brush>
              </colorrole>
             </disabled>
            </palette>
           </property>
           <property name="inputMask">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Имя файла (.txt/.stl)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="StructureLbl">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Представление</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="StructureCombo">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="LoadBtn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Загрузить</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="CamerasLayout">
         <item>
          <widget class="QLabel" name="CamerasLbl">
           <property name="text">
            <string>Камеры</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="CameraCoordsLayout">
           <item>
            <layout class="QVBoxLayout" name="CameraXLayout">
             <item>
              <widget class="QLabel" name="CameraXLbl">
               <property name="text">
                <string>X</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="CameraXSpin"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="CameraYLayout">
             <item>
              <widget class="QLabel" name="CameraYLbl">
               <property name="text">
                <string>Y</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="CameraYSpin"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="CameraZLayout">
             <item>
              <widget class="QLabel" name="CameraZLbl">
               <property name="text">
                <string>Z</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="CameraZSpin"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="AddCameraBtn">
           <property name="text">
            <string>Добавить</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="ObjListLayout">
         <item>
          <widget class="QLabel" name="ObjListLbl">
           <property name="text">
            <string>Объекты</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="ObjList">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="resizeMode">
            <enum>QListView::Fixed</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="ObjButtonsLayout">
         <property name="sizeConstraint">
          <enum>QLayout::SetFixedSize</enum>
         </property>
         <item>
          <widget class="QPushButton" name="DelObjBtn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Удалить выделенное</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="CreateCompositeBtn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Объединить в композит</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="MoveLayout">
         <item>
          <widget class="QLabel" name="MoveX">
           <property name="text">
            <string>X</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="MoveXSpin">
           <property name="minimum">
            <double>-500.000000000000000</double>
           </property>
           <property name="maximum">
            <double>500.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="MoveY">
           <property name="text">
            <string>Y</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="MoveYSpin">
           <property name="minimum">
            <double>-500.000000000000000</double>
           </property>
           <property name="maximum">
            <double>500.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="MoveZ">
           <property name="text">
            <string>Z</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="MoveZSpin">
           <property name="minimum">
            <double>-500.000000000000000</double>
           </property>
           <property name="maximum">
            <double>500.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="MoveBtn">
           <property name="text">
            <string>Переместить</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="ScaleLayout">
         <item>
          <widget class="QLabel" name="ScaleX">
           <property name="text">
            <string>X</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="ScaleXSpin">
           <property name="minimum">
            <double>-500.000000000000000</double>
           </property>
           <property name="maximum">
            <double>500.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="ScaleY">
           <property name="text">
            <string>Y</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="ScaleYSpin">
           <property name="minimum">
            <double>-500.000000000000000</double>
           </property>
           <property name="maximum">
            <double>500.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="ScaleZ">
           <property name="text">
            <string>Z</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="ScaleZSpin">
           <property name="minimum">
            <double>-500.000000000000000</double>
           </property>
           <property name="maximum">
            <double>500.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ScaleBtn">
           <property name="text">
            <string>Масштабировать</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="RotateLayout">
         <item>
          <widget class="QLabel" name="RotateX">
           <property name="text">
            <string>X</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="RotateXspin">
           <property name="minimum">
            <double>-500.000000000000000</double>
           </property>
           <property name="maximum">
            <double>500.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="RotateY">
           <property name="text">
            <string>Y</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="RotateYSpin">
           <property name="minimum">
            <double>-500.000000000000000</double>
           </property>
           <property name="maximum">
            <double>500.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="RotateZ">
           <property name="text">
            <string>Z</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="RotateZSpin">
           <property name="minimum">
            <double>-500.000000000000000</double>
           </property>
           <property name="maximum">
            <double>500.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="RotateBtn">
           <property name="text">
            <string>Повернуть</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGraphicsView" name="Canvas"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="objectMovePushbutton">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>970</y>
      <width>161</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Переместить</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
